hammers/scripts/tests/lease-stacking-test-config.json

```diff
@@ -0,0 +1,15 @@
+{
+    "site": "test",
+    "exclude_projects": [
+        "chameleon",
+        "openstack"
+    ],
+    "lease_coverage_threshold": 0.9,
+    "high_end_node_coverage_threshold": 0.5,
+    "minimum_lease_window_days": 21,
+    "min_nodes_for_coverage": 4,
+    "exclude_node_types": [],
+    "high_end_node_types": [],
+    "sender_email": "noreply@chameleoncloud.org",
+    "manager_email": "help@chameleoncloud.org"
```

Comment:
We shouldn't send to "real" emails during the test!
(just to prevent accidents)